resources/public/api/conf/2.0/application.raml

```diff
@@ -49,6 +49,16 @@ traits:
         type: string
         required: true
         example: application/vnd.hmrc.2.0+json
+  acceptSingleMessageGetHeader:
+    headers:
+      Accept:
+        description: |
+          Specifies the response format, which must be JSON or JSON+XML, and the [version](/api-documentation/docs/reference-guide#versioning) of the API to be used.
+          
+          Acceptable values are "application/vnd.hmrc.2.0+json" or "application/vnd.hmrc.2.0+json-xml"
```

Comment:
Should back ticks be used?
`application/vnd.hmrc.2.0+json`
`application/vnd.hmrc.2.0+json-xml`

End of second sentence needs a full stop.